src/components/ResumeList/ResumeList.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react';
+import { BoxProps } from './type';
+
+import ResumeListDefault from './ResumeListDefault';
+import ResumeListMobile from './ResumeListMobile';
+
+export const ResumeList: FC<BoxProps> = ({ ...props }) => {
```

Comment:
You're using all of the props so there is no need for spreading them. Alternatively, you can take out `mobile` from the rest of the props, as you don't need it inside children components